functions/reacts.py

```diff
@@ -0,0 +1,135 @@
+from enum import StrEnum
+import json
+from pathlib import Path
+from pydantic import BaseModel
+from re import compile, IGNORECASE, UNICODE
+from discord import Message
+from random import choice, sample
+
+from modules.probability import mock_bernoulli
+
+class ReactReplyType(StrEnum):
+    text = "text"
+    image = "image"
+    video = "video"
+    audio = "audio"
+    file = "file"
+
+class ReactCriterias(BaseModel):
+    keywords: list[str]
+    match_whole_word: bool
+    match_link_id: str = ""
+
+class ReactPossibleReaction(BaseModel):
```

Comment:
Should probably add comments to explain what these enums mean

Reply:
i wrote a whole readme :chisobcat: